Functions.Templates/Templates/SendGrid-PowerShell/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+    "defaultFunctionName": "SendGrid",
+    "description": "$SendGrid_description",
+    "name": "SendGrid",
+    "language": "PowerShell",
+    "category": [
+        "$temp_category_samples",
+        "$temp_category_dataProcessing"
+    ],
+    "categoryStyle": "other",
+    "enabledInTryMode": false,
+    "userPrompt": [
+        "to",
+        "from",
+        "subject",
+        "text",
+        "apiKey"
+    ],
+    "extensions": [
+      {
+        "id": "Microsoft.Azure.WebJobs.Extensions.SendGrid",
+        "version": "3.0.0"
+      }

```

Comment:
It needs storage extension as well. Add the following as well

{
        "id": "Microsoft.Azure.WebJobs.Extensions.Storage",
        "version": "3.0.0"
 }

Reply:
Good catch. Added.